### PR TITLE
Configs compilation and jinja templates, renaming

### PR DESCRIPTION
**Pull request by Kuyantsev-VP**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,5 +11,6 @@ RUN pip install -r requirements.txt
 
 ADD main.py ./
 ADD config.py ./
+ADD ./schemas/admin_validation_schema.yaml ./schemas/
```
Comment (rvadim) [inline Dockerfile]:
```
ADD schemas ./schemas
```

File: config_compile.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import yaml
+from jinja2 import FileSystemLoader, Environment
+
+
+def config_compile(config_data, template_name):
+    env = Environment(loader=FileSystemLoader('jinja_templates'),
```
Comment (rvadim) [inline config_compile.py]: Just 'templates'

File: config_compile.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+                      trim_blocks=True,
```
Comment (rvadim) [inline config_compile.py]: Leave this variable defaulted please

**Review by rvadim — CHANGES_REQUESTED** [review]

File: config_compile.py (hunk 3 above)
Comment (Kuyantsev-VP) [reply config_compile.py]: done

File: config_compile.py (hunk 2 above)
Comment (Kuyantsev-VP) [reply config_compile.py]: done

File: Dockerfile (hunk 1 above)
Comment (Kuyantsev-VP) [reply Dockerfile]: done

**Review by rvadim — APPROVED** [review]

**Pull request merged**